packages/codegen/src/generateFactoriesFiles.ts

```diff
@@ -10,6 +10,7 @@ export function generateFactoriesFiles(entities: EntityDbMetadata[]): CodegenFil
   const entityFiles = entities.map(({ entity }) => {
     const name = pascalCase(entity.name);
     const contents = code`
+      /** @ignore */
```

Comment:
The ignores in the codegen files are 100% fine b/c we'd never see them; for the factories, given we "see" this code, I wouldn't mind skipping the ignore here... :thinking: wdyt?

Reply:
My thinking with the factories was without the `@ignore`, you would end up with 100s of entries/pages in the docs. 90+% of which were the vanilla default factories.

Seemed to me that those cluttered documentation with generated cruft. By putting in the `@ignore` in the codegen step, it allows the maintainer to put in real documentation, should that be valuable. I've done that for three factories which have special behaviours in the RP space so far:
![image](https://user-images.githubusercontent.com/24765506/192026743-510f38a8-f196-4d34-b056-cb8d7da7257e.png)

![image](https://user-images.githubusercontent.com/24765506/192026868-905817ab-8cfc-4f08-ac5e-7c742a88b210.png)

Reply:
Fwiw is there an option that docs are opt _in_ instead of opt out? Like only include things in the docs that explicitly have jsdocs?

Granted, our entities do not today ... but, dunno, I think that would be more reasonable even for like the code in our entities, like if we have ~5 methods on the entity, and only 1 of it has docs, then put that one in the docs, and just ignore the others.

Reply:
Maybe https://typedoc.org/guides/options/#excludenotdocumented ?

Reply:
I suspect that option would render the docs pretty useless.

For instance, you would then presumably not see any of the fields which exist on entity, except for those which are defined in the non-codegen file and have an explicit doc comment. So discoverability of things like _oh, an Author has a collection of Books_ would be lost.

Reply:
Though, I guess we could change the direction of this PR from _ignore things_ to _generate semi-reasonable docs where appropriate_ 🤔

Reply:
Ah yeah, agreed we'd want to see those ... but right, probably pretty to flag them in codegen.

Maybe the most annoying would be all of our `Author.ts` files that we'd have to like glob in a `/** The author entity. */` for all of our ~150 some existing entities? Or at least I assume we'd need to add those, otherwise the entity itself wouldn't show up?

...or I guess it would show up by factor of some of its fields being in the docs, so that would get it pulled in? That's maybe not too bad then?